packages/@aws-cdk/aws-apigatewayv2/lib/common/authorizer.ts

```diff
@@ -0,0 +1,33 @@
+import { IResource } from '@aws-cdk/core';
+
+/**
+ * Represents an Authorizer.
+ */
+export interface IHttpAuthorizer extends IResource {
+  /**
+   * Id of the Authorizer
+   * @attribute
+   */
+  readonly authorizerId: string
+
+  /**
+   * A human friendly name for this Authorizer
+   * @attribute
+   */
+  readonly authorizerName: string
```

Comment:
Authorizer name is not a return type from CloudFormation, so it should not be marked `@attribute`.

I would suggest dropping this entirely unless you have a use case for it.